scipy/stats/stats.py

```diff
@@ -207,7 +207,6 @@
            'combine_pvalues', 'wasserstein_distance', 'energy_distance',
            'brunnermunzel', 'epps_singleton_2samp']
 
-
```

Comment:
Don't remove this blank line.  From PEP 8: "Surround top-level function and class definitions with two blank lines."